Simon-Initiative/oli-torus
# Overview

Overview page.

Test:
- Add a collaborator by email
- Add the same collaborator (should fail)
- Remove a collaborator
- Remove the project owner (should fail)
- Edit project description

Changes:
- Add controller and update logic for collaborators (authors_projects)
- Add uniqueness constraint for authors_projects
- Add update logic for projects
